pkgs/applications/misc/bemenu/default.nix

```diff
@@ -20,6 +20,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-U4IMfDvQ0rfEJhE3Uext2c/Cs0mjy1tw+k8uk441Ag8=";
   };
 
+  patches = [
+    # Pull fix pending upstream inclusion to fix build against ncurses-6.3:
+    #  https://github.com/Cloudef/bemenu/pull/211
+    (fetchpatch {
+      name = "ncurses-6.3";
+      url = "https://github.com/Cloudef/bemenu/commit/9e7ebd3bf68ccaa10f6856be0a905114d1c9962e.patch";
```

Comment:
Looks like upstream did a rebase-and-merge, would you mind changing this to the mainline commit https://github.com/Cloudef/bemenu/commit/d31164db756989579468946aba62969e42c7ed28

Reply:
Sounds good. Updated patch link to upstreamed version and added `.patch` suffix to `name =` while at it.